src/components/TradeAssetSearch/TradeAssetSearch.tsx

```diff
@@ -100,9 +102,15 @@ export const TradeAssetSearch: FC<TradeAssetSearchProps> = ({
 
   const popularAssets = useMemo(() => {
     const unfilteredPopularAssets = popularAssetsByChainId?.[activeChainId] ?? []
-    if (allowWalletUnsupportedAssets) return unfilteredPopularAssets
+    if (allowWalletUnsupportedAssets || !isConnected) return unfilteredPopularAssets
```

Comment:
Allows popular assets to display instead of blank perma-loading state:

![image](https://github.com/user-attachments/assets/d0c9c70c-e6df-42d9-b9a2-c185e08d5fe5)
![image](https://github.com/user-attachments/assets/d13f9220-957b-4596-bc35-50c4bb3075ae)